<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundManagerWindow</class>
 <widget class="QDialog" name="SoundManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Speaker</string>
   </property>
   <widget class="QSlider" name="speakerSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="displaySpeaker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Microphone</string>
   </property>
   <widget class="QSlider" name="microphoneSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="displayMicrophone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>100</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveBtn</sender>
   <signal>accepted()</signal>
   <receiver>SoundManagerWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>rejected()</signal>
   <receiver>SoundManagerWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
